docs/Options.md

```diff
@@ -16,38 +16,39 @@ The default order is:
 
 ```zsh
 SPACESHIP_PROMPT_ORDER=(
-  time          # Time stampts section
-  user          # Username section
-  dir           # Current directory section
-  host          # Hostname section
-  git           # Git section (git_branch + git_status)
-  hg            # Mercurial section (hg_branch  + hg_status)
-  package       # Package version
-  node          # Node.js section
-  ruby          # Ruby section
-  elixir        # Elixir section
-  xcode         # Xcode section
-  swift         # Swift section
-  golang        # Go section
-  php           # PHP section
-  rust          # Rust section
-  haskell       # Haskell Stack section
-  julia         # Julia section
-  docker        # Docker section
-  aws           # Amazon Web Services section
-  venv          # virtualenv section
-  conda         # conda virtualenv section
-  pyenv         # Pyenv section
-  dotnet        # .NET section
-  ember         # Ember.js section
-  kubecontext   # Kubectl context section
-  exec_time     # Execution time
-  line_sep      # Line break
-  battery       # Battery level and status
-  vi_mode       # Vi-mode indicator
-  jobs          # Backgound jobs indicator
-  exit_code     # Exit code section
-  char          # Prompt character
+  time            # Time stampts section
+  user            # Username section
+  dir             # Current directory section
+  host            # Hostname section
+  git             # Git section (git_branch + git_status)
+  git_commit_time  # Display time since previous commit
```

Comment:
Could you move all comments further away to the right, by like 5-10 spaces? So that people don't have to align this every time they make a new section with a longer name?